assets/javascripts/admin_groups.js

```diff
@@ -273,35 +272,37 @@ function saveReorganizedGroups() {
         $('#reorganize_groups_error').show();
         $('#reorganize_groups_progress').hide();
         $('#reorganize_groups_error_message').text(thrownError ? thrownError : 'something went wrong');
-        $('html, body').animate({scrollTop: 0}, 1000);
+        $('html, body').animate({
+            scrollTop: 0
+        }, 1000);
```

Comment:
The previous  style was better.

Reply:
Added the CLI option `--brace-style=collapse,preserve-inline` to preserve the style without line breaks. It addes a bit of spaces though, I hope that's ok:

```
-        $('html, body').animate({scrollTop: 0}, 1000);
+        $('html, body').animate({ scrollTop: 0 }, 1000);
```